PrimeiroPrograma/ContaCorrente.cs

```diff
@@ -13,6 +13,24 @@ public ContaCorrente(Cliente Titular, int numeroDaconta, double saldo) : base(Ti
         }
         public double TaxaCorrente { get; set; }
 
+        public double consultaSaldo()
+        {
+            return this.Saldo;
+        }
+
+        public void Depositar(double valorDposito)
+        {
+            if (valorDposito > 0)
+            {
+                this.Saldo += valorDposito;
+                Console.WriteLine("Deposito realizado com sucesso! \n\nSaldo atual é de: " + this.Saldo);
+            }
+            else if (valorDposito == 0)
```

Comment:
voce pode tirar esse else if (valorDposito == 0) e colocar apenas o else. Pois se o valorDposito for menor que 0 ele pula esse if
algo como:
if (valorDposito > 0)
            {
                this.Saldo += valorDposito;
                Console.WriteLine("Deposito realizado com sucesso! \n\nSaldo atual é de: " + this.Saldo);
            }
else
{
  Console.WriteLine("Voce nao pode depositar o valor: " + valorDposito);
}